fmt/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "forge-fmt"
+version = "0.2.0"
+edition = "2021"
+authors = ["Foundry Contributors"]
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/foundry-rs/foundry"
+repository = "https://github.com/foundry-rs/foundry"
```

Comment:
Is this intentional? Might be misleading for example if we want to publish it on crates.io but I guess this isn't supposed to be published?

Reply:
As we're vendoring for now, let's leave it as is